ui/apps/dashboard/src/layout/sidebar.tsx

```diff
@@ -25,10 +30,15 @@ const Sidebar = () => {
       .map((m) => (m.handle as IRouteObjectHandle).sidebarKey);
   }, [matches]);
   return (
-    <div className={'w-full h-full overflow-y-auto'}>
+    <div
+      className={cn('w-full', 'h-full', 'overflow-y-auto', {
+        'collapsed-sidebar': collapsed,
```

Comment:
It seems that `collapsed-sidebar` class doesn't have any effect 😃 
Last time, i did not mention this, may it takes effect, can u explain it?

Reply:
Oh, the `Menu` component from `ant design` already handles the translation. Which is redundant. Thanks a lot.